Makefile

```diff
@@ -34,6 +34,7 @@ upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -qr requirements/pip-tools.txt
 	# Make sure to compile files after any other files they include!
+	$(PIP_COMPILE) --allow-unsafe -o requirements/pip.txt requirements/pip.in
```

Comment:
Right now, `PIP_COMPILE` includes `--rebuild`; we can add that parameter to just this line, and then remove it from that variable.  We should only need it for the first pip-compile run in each upgrade target.